03 Writing Algorithms/31 Machine Learning/03 Popular Libraries/10 XGBoost/02 Import Libraries.html

```diff
@@ -2,8 +2,10 @@
 
 <div class="section-example-container">
     <pre class="python">from AlgorithmImports import *
+# Import xgboost model
 import xgboost as xgb
```

Comment:
Please don't comment non-QC API methods. And generally only comment more significant code blocks (no need to comment anything less than 20 characters)